tests/Oryx.RuntimeImage.Tests/Php-fpm/PhpFpmImageTest.cs

```diff
@@ -169,5 +169,30 @@ public void Redis_IsInstalled(string imageTag)
                 result.GetDebugInfo());
 
         }
+
```

Comment:
I see that in the previous PR, https://github.com/microsoft/Oryx/pull/1547/files, some tests were added to validate the change change for Imagick (tests/Oryx.Integration.Tests/Php/PhpImagickExampleTest.cs). I would expect these tests to fail given that the runtime images were not regenerated. Just asking in case you already know: would you have an idea of why this works?

Reply:
The tests that were added were for the PHP images (not the PHP FPM images), which did have their version-specific Dockerfiles updated in that case.

Reply:
I agree the tests were added in the "PHP" section, but they did contain calls to methods such as "PhpFpmImagickExampleAsync". I would have expected this to fail